src/panels/lovelace/cards/hui-picture-entity-card.js

```diff
@@ -0,0 +1,165 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import './hui-error-card.js';
+import '../../../components/ha-card.js';
+
+import { STATES_OFF } from '../../../common/const.js';
+import computeDomain from '../../../common/entity/compute_domain.js';
+import computeStateDisplay from '../../../common/entity/compute_state_display.js';
+import computeStateDomain from '../../../common/entity/compute_state_domain.js';
+import computeStateName from '../../../common/entity/compute_state_name.js';
+import createErrorCardConfig from '../common/create-error-card-config.js';
+
+import LocalizeMixin from '../../../mixins/localize-mixin.js';
+
+const OFFLINE = 'Offline';
+
+/*
+ * @appliesMixin LocalizeMixin
+ */
+class HuiPictureEntityCard extends LocalizeMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style>
+        ha-card {
+          position: relative;
+          cursor: pointer;
+          overflow: hidden;
+        }
+        img {
+          display: block;
+          width: 100%;
+          height: auto;
+        }
+        .box {
+          @apply --paper-font-common-nowrap;
+          position: absolute;
+          left: 0;
+          right: 0;
+          bottom: 0;
+          background-color: rgba(0, 0, 0, 0.3);
+          padding: 16px;
+          font-size: 16px;
+          line-height: 16px;
+          color: white;
+          display: flex;
+          justify-content: space-between;
+        }
+        #title {
+          font-weight: 500;
+        }
+      </style>
+
+      <ha-card on-click="_cardClicked">
+        <img id="image" src="">
+        <div class="box">
+          <div id="title"></div>
+          <div id="state"></div>
+        </div>
+        <template is="dom-if" if="[[_error]]">
+          <hui-error-card config="[[_error]]"></hui-error-card>
```

Comment:
This is weird, this renders a card inside another card? This is exactly why I want a `setConfig(config)` method that can raise, so we can _replace_ a card with invalid config